src/components/navbar/navbar.component.tsx

```diff
@@ -4,17 +4,25 @@ import UserMenuPanel from "../navbar-header-panels/user-menu-panel.component";
 import SideMenuPanel from "../navbar-header-panels/side-menu-panel.component";
 import Logo from "../logo/logo.component";
 import { navigate, ExtensionSlot } from "@openmrs/esm-framework";
+import AppSwitcher20 from "@carbon/icons-react/lib/app-switcher/20";
+import Close20 from "@carbon/icons-react/lib/close/20";
 import {
   HeaderContainer,
   Header,
   HeaderMenuButton,
   HeaderName,
   HeaderGlobalBar,
-  HeaderGlobalAction
+  HeaderGlobalAction,
+  HeaderPanel,
+  Switcher,
+  SwitcherItem,
+  SwitcherDivider
 } from "carbon-components-react/es/components/UIShell";
 import { LoggedInUser, UserSession } from "../../types";
 import styles from "./navbar.scss";
 import { getCurrentSession } from "../../root.resource";
+import SourceFiles from "../../urls";
+import { HeaderSideNavItems } from "carbon-components-react";
```

Comment:
Use selective imports as shown above for the UIShell. Don't use the full package.